src/main/java/com/imaee/propinq/buildings/controllers/implementations/BuildingController.java

```diff
@@ -0,0 +1,27 @@
+package com.imaee.propinq.buildings.controllers.implementations;
+
+import com.imaee.propinq.buildings.controllers.interfaces.IBuildingController;
+import com.imaee.propinq.buildings.controllers.responses.BuildingResponse;
+import com.imaee.propinq.buildings.services.interfaces.IBuildingService;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BuildingController implements IBuildingController {
+
+    private final IBuildingService buildingService;
+    public BuildingController(IBuildingService buildingService) {
+        this.buildingService = buildingService;
+    }
```

Comment:
Ya que estamos usando Lombok, no hace falta agregar los constructores a mano (son medio toscos y si agregamos un nuevo atributo vamos a tener que cambiarlos). Reemplazalo por la anotación de `@AllArgsConstructor`

```suggestion
@AllArgsConstructor
public class BuildingController implements IBuildingController {

    private final IBuildingService buildingService;
    
```

Entiendo que a ese cambio ya lo había hecho crous igual